### PR TITLE
chore: create performance metrics project

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Basic setup of Performance Metrics project

Closes https://opentrons.atlassian.net/browse/EXEC-380

# Test Plan

- [x] Verify all makefile commands can run successfully (GH Actions will be in another PR)
- [x] Build wheel, install, and verify it runs

### CODE REVIEW

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Let's keep the tooling we use in monorepo python projects consistent, with flake8, mypy, black, and so on at the same versions as the other projects. This eases everybody's ability to context switch.

File: performance-metrics/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[tool.ruff]
```
Comment (sfoster1) [inline performance-metrics/pyproject.toml]: need to remove this one too - we're still using `setup.py`

**Review by sfoster1 — CHANGES_REQUESTED** [review]

File: performance-metrics/pyproject.toml (hunk 1 above)
Comment (DerekMaggio) [reply performance-metrics/pyproject.toml]: https://github.com/Opentrons/opentrons/pull/14806/commits/71cb58415df045e67a42063338f3080048836b43 should fix it

**Review by sfoster1 — APPROVED** [review]
lgtm!

**Pull request merged**